testgrid/config.yaml

```diff
@@ -4449,6 +4451,8 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-gci-gce-autoscaling-migs
   - name: gci-gke-autoscaling
     test_group_name: ci-kubernetes-e2e-gci-gke-autoscaling
+  - name: gci-gke-autoscaling-gpu
+    test_group_name: ci-kubernetes-e2e-gci-gke-autoscaling-gpu
```

Comment:
Add this in the GPU testgrid tab as well.

Reply:
@mindprince Sure. Which tab you have in mind exactly. I do not see anything with gpu/accel in its name. `wg-resource-management`?

Reply:
Yes. `wg-resource-management`

Reply:
Done. I did not add any alerts though.